DfE.Hexagonal.Driver.Application.Tests/Tests/School/GetAllSchoolTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using DfE.Hexagonal.Application.UseCases.School.GetAllSchools;
+using DfE.Hexagonal.Driver.Application.Tests.TestDoubles;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DfE.Hexagonal.Driver.Application.Tests.Tests.School
+{
+    public class GetAllSchoolTests
+    {
+        [Fact]
+        public async void GetAllSchools_returns_a_list_of_schools()
+        {
+            // arrange
+
+            var repository = SchoolRepositoryTestDoubles.MockFor(new List<SchoolOutput>()
+            {
+                new SchoolOutput()
+                {
+                    Id = Guid.NewGuid(),
+                    Name = "test",
+                    Description = "test",
+                    OpenDate = DateTime.Now
+                },
+                new SchoolOutput()
+                {
+                    Id = Guid.NewGuid(),
+                    Name = "test2",
+                    Description = "test2",
+                    OpenDate = DateTime.Now
+                }
+            });
+
+            // act
+
+            var sut = new GetAllSchools(repository);
+            var result = await sut.Handle();
+
+            // assert
+
+            Assert.Equal(2, result.Schools.Count());
```

Comment:
If we are testing that a list of schools is being returned (per the test name), let's assert/validate that it is a list of schools being returned 😄 

For example:
`Assert.IsType<List<SchoolOutput>>(result.Schools);`

_(Catherine (GitHub won't seem to let me tag her) taught me this XUnit assertion, thank you!)_

Reply:
The tests need improvement - this is a good one.